internal/backend/temporalclient/config.go

```diff
@@ -38,6 +38,8 @@ type Config struct {
 	DevServer testsuite.DevServerOptions `koanf:"dev_server"`
 	TLS       tlsConfig                  `koanf:"tls"`
 
+	DataConverter DataConverterConfig `koanf:"data_converter"`
```

Comment:
do we want to add enableDataConverter to make stuff easier for open source ?
we don't really care about it in this case

Reply:
not sure the relation to oss? if encryption and compression is not enabled it's just a null converter

Reply:
oh, so it will work with default configuration without any values ?

Reply:
see comment below.